arbitrum-docs/welcome/get-started.md

```diff
@@ -0,0 +1,80 @@
+| [A gentle introduction to Arbitrum](/for-devs/gentle-introduction-dapps) | A technical introduction to Arbitrum's suite of scaling solutions.                                       |
+| [Quickstart (Solidity)](/for-devs/quickstart-solidity-hardhat)           | Targeted at web2 developers who want to deploy their first Solidity smart contract to Arbitrum.          |
+| [Quickstart (Rust)](/stylus/stylus-quickstart)                           | Targeted at web3 developers who want to deploy their first Rust smart contract to Arbitrum using Stylus. |
+
+
+## Arbitrum for node runners
+
+**Node runners** run the machines that support the Arbitrum ecosystem.
+
+| Resource                                                                                                  | Description                                                                                                           |
+| --------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| [Run a full node](/node-running/how-tos/running-a-full-node)                                              | Targeted at node runners who want to access Arbitrum chains without having to connect to a third-party node.          |
+| [Configure a Data Availability Committee](/node-running/how-tos/data-availability-committee/introduction) | Targeted at Data Availability Committee members and Orbit chain operators who want to run a Data Availability Server. |
+
+
+## Arbitrum for chain operators
+
+**Chain operators** use Arbitrum Orbit to run special-purpose Rollup and AnyTrust chains.
+
+| Resource                                                                   | Description                                                                                     |
+| -------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| [Orbit gentle introduction](/launch-orbit-chain/orbit-gentle-introduction) | Targeted at readers who want to understand Orbit's value proposition and use cases.             |
+| [Orbit quickstart](/launch-orbit-chain/orbit-quickstart)                   | Targeted at chain operators who want to deploy their first Arbitrum chain using Arbitrum Orbit. |
+
+
+
+## How it works
+
+| Resource                                                                                           | Description                                       |
+| -------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
+| [Arbitrum whitepaper](https://github.com/OffchainLabs/nitro/blob/master/docs/Nitro-whitepaper.pdf) | The original whitepaper that introduced Nitro.    |
```

Comment:
I guess this is in alphabetical order, but if it's ok to reorder, I'd place Arbitrum Nitro first (most referenced page), then Anytrust, then the whitepaper and finally the DAO docs.